.github/workflows/test_dbt_models.yaml

```diff
@@ -23,3 +23,10 @@ jobs:
         run: dbt test --target "$TARGET"
         working-directory: ${{ env.PROJECT_DIR }}
         shell: bash
+
+      - name: Send failure notification
+        if: failure()
```

Comment:
I _think_ this works and runs only if the workflow fails. I found the docs a little confusing here. Will test it more once it's merged and I can run workflow dispatch manually.